macros/cross_db_utils/datediff.sql

```diff
@@ -3,15 +3,15 @@
 {% endmacro %}
 
 
-{% macro default__datediff(first_date, second_date, datepart) %}
+{%- macro default__datediff(first_date, second_date, datepart) -%}
```

Comment:
The leading `-` doesn't do anything (there is no whitespace prior to it to remove) - this is also true for the trailing `-` in the `endmacro` block, but I haven't added this suggestion in every place
```suggestion
{% macro default__datediff(first_date, second_date, datepart) -%}
```